packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -25,8 +27,28 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>>
   protected root = React.createRef<HTMLDivElement>();
   protected input = React.createRef<Autofill>();
   protected selection: Selection;
-  protected floatingPickerProps: IBaseFloatingPickerProps<T>;
-  protected selectedItemsListProps: IBaseSelectedItemsListProps<T>;
+
+  public static getDerivedStateFromProps(newProps: IBaseExtendedPickerProps<any>) {
+    const updatedState: Partial<IBaseExtendedPickerProps<any>> = {};
+    let hasUpdate: boolean = false;
+
+    if (newProps.floatingPickerProps) {
+      updatedState.floatingPickerProps = newProps.floatingPickerProps;
+      hasUpdate = true;
+    }
+
+    if (newProps.selectedItemsListProps) {
+      updatedState.selectedItemsListProps = newProps.selectedItemsListProps;
+      hasUpdate = true;
+    }
+
+    if (newProps.selectedItems) {
```

Comment:
As far as I can tell, `selectedItems` is never updated in state--so could you just get rid of the state member (and read directly from props), or would that break child classes?

Reply:
I believe that it might break child classes which is part of why I'm doing the weird getter hybrid with props. It's not ideal but I don't know how to go forward without it being a breaking change in some other way.